packages/tailwindcss/src/design-system.ts

```diff
@@ -45,6 +47,12 @@ export function buildDesignSystem(theme: Theme): DesignSystem {
     utilities,
     variants,
 
+    // How to mark important utilities
+    // - wrap with a selector (any string)
+    // - add an !important (true)
+    // - do nothing (false)
```

Comment:
I think we can improve this comment a bit. Took a stab at it, but not sure if it is much better:

```suggestion
    // Strategies:
    // - `string` — wrap each rule with the provided selector
    // - `true`   — add `!important` to every declaration
    // - `false`  — do nothing
```

Reply:
I ended up removing this comment after splitting them